beacon-chain/rpc/prysm/v1alpha1/validator/BUILD.bazel

```diff
@@ -116,69 +182,19 @@ go_test(
-        "//beacon-chain/forkchoice/protoarray:go_default_library",
-        "//beacon-chain/operations/attestations:go_default_library",
-        "//beacon-chain/operations/slashings:go_default_library",
-        "//beacon-chain/operations/synccommittee:go_default_library",
-        "//beacon-chain/operations/voluntaryexits:go_default_library",
-        "//beacon-chain/p2p/testing:go_default_library",
-        "//beacon-chain/state:go_default_library",
-        "//beacon-chain/state/stategen:go_default_library",
-        "//beacon-chain/state/stategen/mock:go_default_library",
-        "//beacon-chain/state/v1:go_default_library",
-        "//beacon-chain/state/v3:go_default_library",
-        "//beacon-chain/sync/initial-sync/testing:go_default_library",
-        "//config/fieldparams:go_default_library",
-        "//config/params:go_default_library",
-        "//consensus-types/blocks:go_default_library",
-        "//consensus-types/interfaces:go_default_library",
-        "//consensus-types/primitives:go_default_library",
-        "//container/trie:go_default_library",
-        "//crypto/bls:go_default_library",
-        "//encoding/bytesutil:go_default_library",
-        "//encoding/ssz:go_default_library",
-        "//proto/engine/v1:go_default_library",
-        "//proto/eth/v1:go_default_library",
-        "//proto/prysm/v1alpha1:go_default_library",
-        "//proto/prysm/v1alpha1/attestation:go_default_library",
-        "//proto/prysm/v1alpha1/attestation/aggregation/attestations:go_default_library",
-        "//testing/assert:go_default_library",
-        "//testing/mock:go_default_library",
-        "//testing/require:go_default_library",
-        "//testing/util:go_default_library",
-        "//time:go_default_library",
-        "//time/slots:go_default_library",
-        "@com_github_d4l3k_messagediff//:go_default_library",
-        "@com_github_ethereum_go_ethereum//common:go_default_library",
-        "@com_github_ethereum_go_ethereum//core/types:go_default_library",
-        "@com_github_golang_mock//gomock:go_default_library",
-        "@com_github_pkg_errors//:go_default_library",
-        "@com_github_prysmaticlabs_go_bitfield//:go_default_library",
-        "@com_github_sirupsen_logrus//:go_default_library",
-        "@com_github_sirupsen_logrus//hooks/test:go_default_library",
-        "@org_golang_google_grpc//codes:go_default_library",
-        "@org_golang_google_grpc//status:go_default_library",
-        "@org_golang_google_protobuf//proto:go_default_library",
-        "@org_golang_google_protobuf//types/known/emptypb:go_default_library",
+    eth_network = "minimal",
+    tags = ["minimal"],
+    deps = common_deps,
+)
+
+go_test(
+    name = "go_mainnet_test",
+    timeout = "moderate",
+    srcs = [
+        "attester_mainnet_test.go",
+        "server_mainnet_test.go",
+        "status_mainnet_test.go",
     ],
+    embed = [":go_default_library"],
+    deps = common_deps,
 )
```

Comment:
Some tests need to run using mainnet config. I separated these tests out to new `mainnet` files and put them under a new rule